kfdefs/overlays/moc/smaug/opf-dashboard/route.enc.yaml

```diff
@@ -0,0 +1,47 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+    name: odh-dashboard-google
```

Comment:
I don't think there's need for this route, because:

1. There's no secret generator importing it
2. Instead there's the same resource deployed via `route.yaml` with dynamic certificate.

Please remove the `route.enc.yaml` file